packages/cf.js/test/fixture.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  AppInstanceInfo,
+  AssetType,
+  INodeProvider,
+  Node
+} from "@counterfactual/common-types";
+import { BigNumber } from "ethers/utils";
```

Comment:
@snario Why? `import` actually makes more sense. You are importing a type, not destructuring an object (albeit in this context it might mean the same).

Reply:
yeah let's not use destructing assignment when we should be using modules

Reply:
I agree with @joelalejandro in the case of types but I disagree in the case of util functions. In my current PR I have code like this:

```js
const {
  keccak256,
  solidityPack,
  defaultAbiCoder,
  SigningKey,
  Interface,
  bigNumberify,
  hexlify,
  randomBytes,
  getAddress
} = ethers.utils;
```

It would be annoying to do an import for all of them, no?

Reply:
@snario you can do exactly the same:

```import {
  keccak256,
  solidityPack,
  defaultAbiCoder,
  SigningKey,
  Interface,
  bigNumberify,
  hexlify,
  randomBytes,
  getAddress
} from "ethers/utils";
```

Why would we deal with this in a different way? They're exported members of a module, regardless of them being types or functions.

Reply:
I agree. This is weird to me to suggest otherwise

Reply:
Sounds good to me

Reply:
I too prefer import over deconstruction. 

Unfortunately, for some reason we have a lint rule that disallows imports from "ethers" and imports from "ethers/utils" together in the same file. I don't really see a reason to keep it tbh